x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_status_state.tsx

```diff
@@ -348,11 +360,22 @@ const isJobConfigurationConsistent = (
       return (
         jobConfiguration &&
         jobConfiguration.bucketSpan === sourceConfiguration.bucketSpan &&
-        jobConfiguration.indexPattern === sourceConfiguration.indexPattern &&
+        jobConfiguration.indexPattern &&
+        isIndexPatternSubset(jobConfiguration.indexPattern, sourceConfiguration.indexPattern) &&
         jobConfiguration.timestampField === sourceConfiguration.timestampField
       );
     });
 
+const isIndexPatternSubset = (indexPatternSubset: string, indexPatternSuperset: string) => {
+  const subsetSubPatterns = indexPatternSubset.split(',');
+  const supersetSubPatterns = new Set(indexPatternSuperset.split(','));
+
+  return subsetSubPatterns.every(subPattern => supersetSubPatterns.has(subPattern));
+};
+
+const hasError = <Value extends any>(value: Value): value is MandatoryProperty<Value, 'error'> =>
```

Comment:
Maybe a noob question: what does `extends any` do here?

Reply:
It's required to disambiguate the syntax for the typescript compiler. Otherwise it tries to interpret `<Value>` as a JSX factory call.